api/api.js

```diff
@@ -19,14 +19,16 @@ var debug = require('../tools/debug');
 var routes = require('./routes.js');
 
 var PRIVATE = require('../PRIVATE.json');
-var maestro = require('./maestro')(PRIVATE.token); // API-side mqtt client
+
+// Creating API-side MQTT client: maestro !
+var maestro = require('./maestro')(PRIVATE.token);
 
 var server = new http.Server(app);
 var io = require('socket.io')(server);
 
-
 var PORT = 4000;
 
+var sockets = [];
```

Comment:
The best way to broadcast using socket.io is to use [io.sockets.emit()](https://github.com/socketio/socket.io#serveremit)